### PR TITLE
feat: dsa username/password from env vars

### PR DESCRIPTION
**Pull request by raylim**

### CODE REVIEW

File: src/luna/pathology/cli/dsa_annotation_etl.py [hunk 1]
```diff
@@ -29,28 +30,25 @@ def cli(
     dsa_endpoint: str = "???",
     collection_name: str = "???",
     annotation_name: str = "???",
-    username: str = "???",
-    password: str = "???",
+    username: str = "${oc.env:DSA_USERNAME}",
```
Comment (tomp) [inline src/luna/pathology/cli/dsa_annotation_etl.py:33]: Shouldn't this be

    username: str = os.getenv("DSA_USERNAME", "???"),
    password: str = os.getenv("DSA_PASSWORD", "???"),

I'm not sure what oc.env is, but this is the normal pythonic way to use environment variable values.

File: src/luna/pathology/cli/dsa_upload.py [hunk 2]
```diff
@@ -14,16 +14,15 @@
...
 @timed
 @save_metadata
 def cli(
     dsa_endpoint_url: str = "???",
     annotation_file_urlpath: str = "???",
     collection_name: str = "???",
     image_filename: str = "???",
-    username: str = "???",
-    password: str = "???",
+    username: str = "${oc.env:DSA_USERNAME}",
```
Comment (tomp) [inline src/luna/pathology/cli/dsa_upload.py:24]: Same comment as above

**Review by tomp — COMMENTED** [review]
This looks good, except for the way the env vars are being accessed.

File: src/luna/pathology/cli/dsa_annotation_etl.py (hunk 1 above)
Comment (raylim) [reply src/luna/pathology/cli/dsa_annotation_etl.py:33]: It's the way to do it using OmegaConf which is the library we're using for configuration. See https://omegaconf.readthedocs.io/en/2.3_branch/custom_resolvers.html#built-in-resolvers

Comment (raylim) [reply src/luna/pathology/cli/dsa_annotation_etl.py:33]: You can also set it to fallback to ??? but the point of that is that it throws an error if missing, which it no longer does after the resolution.

Comment (tomp) [reply src/luna/pathology/cli/dsa_annotation_etl.py:33]: Ok, makes sense.

**Review by armaank — APPROVED** [review]
This is a good change IMO. In general, it's good to use env variables for this kind of thing, let's be sure to use one-off or temporary tokens and not save actual passwords as .env variables

**Pull request merged**